gitops/octopus/ImageTagRelease.md

```diff
@@ -191,3 +227,7 @@ When your Release runs successfully, you have the option to promote to the next
 **Note**: You will need to prepare your `qa` environment before deploying if you have not already done so. This means that you should have Deployment Targets specified and labeled `qa` if used in your `qa` environment.
 
 ![](images/octo-dep-targets-qa.png)
+
+## TeamCity
+
+The TeamCity [image tag release](https://github.com/microsoft/bedrock/blob/master/gitops/teamcity/ImageTagRelease.md) pipeline works hand in hand with this Octopus pipeline. With the [manifest generation pipeline](https://github.com/microsoft/bedrock/blob/master/gitops/octopus/OctopusDeploy.md), you should be able to complete the entire GitOps CI/CD workflow using TeamCity and Octopus Deploy.
```

Comment:
Maybe we should add a link to the [octopus connection readme](https://github.com/microsoft/bedrock/blob/master/gitops/teamcity/ConnectToOctopus.md) as well?